storage/Devices/BlkDeviceImpl.cc

```diff
@@ -1,5 +1,7 @@
 
 
+#include <iostream>
```

Comment:
https://ci.opensuse.org/job/libstorage-ng-master/7/console
But that failure is because of a ``` cout << cmd_line << endl;```. Do we really want to write to cout in the library?